tools/dev/src/main/groovy/intellijRunConfig.groovy

```diff
@@ -57,13 +58,22 @@ Map controllerEnv = null
 Map invokerEnv = null
 
 containerNames.each{cn ->
+
     //Inspect the specific container
     def inspectResult = "docker inspect $cn".execute().text
     def json = new JsonSlurper().parseText(inspectResult)
 
     def imageName = json[0].'Config'.'Image'
     if (imageName.contains("controller") || imageName.contains("invoker")){
-        def mappedPort = json.'NetworkSettings'.'Ports'.'8080/tcp'[0][0].'HostPort'
+        // extract all the ports exposed by this container
+        def hostPorts = []
+        def mappedPort = json[0].'NetworkSettings'.'Ports'.each{ port, portValue ->
```

Comment:
Need not initialize `mappedPort` here as its being overridden later in line 76

Reply:
thanks ! done.